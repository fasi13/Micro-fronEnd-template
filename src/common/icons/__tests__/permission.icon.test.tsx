import { render } from '@testing-library/react';
import React from 'react';
import { PermissionIcon } from '../index';

test('It should render permissionIcon', async () => {
	const size = { width: 512, height: 512 };

	expect(
		render(
			<PermissionIcon className="" width={size.width} height={size.height} />,
		),
	).toMatchSnapshot();
});
