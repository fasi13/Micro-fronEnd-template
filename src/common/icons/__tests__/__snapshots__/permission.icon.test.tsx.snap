// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render permissionIcon 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root"
        fill="#6D899E"
        focusable="false"
        height="512"
        viewBox="0 0 480 480"
        width="512"
        x="0px"
        xmlns="http://www.w3.org/2000/svg"
        y="0px"
      >
        <g>
          <path
            d="m113.572 338.626c-2.93-2.928-7.678-2.928-10.607 0l-19.88 19.88c-2.929 2.93-2.929 7.678 0 10.607l80.067 80.068c1.407 1.406 3.314 2.196 5.304 2.196s3.896-.79 5.304-2.196l19.881-19.881c1.406-1.407 2.196-3.314 2.196-5.304s-.79-3.896-2.196-5.304zm54.884 94.645-69.461-69.461 9.273-9.273 69.462 69.461z"
          />
          <path
            d="m282.902 243.491-50.435 50.436c-2.929 2.93-2.929 7.678 0 10.607 1.465 1.464 3.385 2.196 5.304 2.196s3.839-.732 5.304-2.196l50.435-50.436c2.929-2.93 2.929-7.678 0-10.607-2.93-2.928-7.678-2.928-10.608 0z"
          />
          <path
            d="m467.067 72.911c.137-4.646-4.131-8.354-8.669-7.629l-76.791 12.19c-1.563.248-3.008.984-4.127 2.104 0 0-167.323 167.181-168.032 167.637v-89.392c0-4.143-3.357-7.5-7.5-7.5h-30.529v-13.805c0-16.834 13.695-30.529 30.529-30.529h22.28l17.389 17.436c10.651 10.679 28.337 10.485 38.747-.442l16.19-16.993h23.689c4.847 0 10.206-.62 10.725-6.729.55-6.493-5.546-8.271-10.725-8.271h-24.109c8.347-8.719 13.492-20.527 13.492-33.522v-29.393c-.001-15.479-12.595-28.073-28.075-28.073h-40.912c-15.48 0-28.074 12.594-28.074 28.073v.024.008 29.36c0 12.727 5.025 24.666 13.467 33.522h-24.084c-25.104 0-45.529 20.425-45.529 45.529v98.506c-10.916-.595-22.223 2.779-30.079 10.635l-62.25 62.258c-25.546 25.538-25.546 68.301-.001 93.849l66.41 66.41c11.509 11.509 27.545 18.605 43.818 19.362.003 13.49 10.979 24.464 24.47 24.464h124.622c21.628 0 33.365-28.555 16.118-42.879l-11.784-9.893v-152.027c17.325 8.387 38.028-4.723 38.028-23.881v-47.445c2.605.859 5.586.259 7.657-1.814 2.866-2.868 2.869-7.734-.005-10.606-7.919-7.911-7.911-21.119.003-29.03 7.907-7.909 21.115-7.909 29.022-.001 2.929 2.93 7.678 2.929 10.606.001 2.867-2.867 2.875-7.733.001-10.606-8.002-8.003-7.997-21.026.01-29.033 7.903-7.905 21.108-7.913 29.011-.003 4.233 4.239 11.78 1.748 12.713-4.125l12.155-76.566c.074-.387.111-.791.123-1.181zm-197.564 39.723c-5.7 4.92-11.455 4.986-17.266.197l-6.825-6.844h30.424zm-19.873-23.652c-13.198-4.796-22.065-17.462-22.065-31.518v-16.01c8.131 4.036 19.806 8.172 33.53 8.172 13.725 0 25.399-4.136 33.53-8.172v16.011c0 22.735-23.692 39.246-44.995 31.517zm-8.991-73.982h40.912c5.978 0 11.021 4.038 12.572 9.526-5.599 3.573-17.897 10.1-33.028 10.1s-27.429-6.527-33.028-10.1c1.551-5.488 6.593-9.526 12.572-9.526zm-46.192 150.321v117.998c0 6.35-5.165 11.515-11.515 11.515s-11.515-5.165-11.515-11.515v-117.998zm-5.069 321.183c12.197-2.242 23.656-7.908 32.812-16.279h31.408v26.775h-54.811c-5.222 0-9.409-5.274-9.409-10.496zm97.224-80.585c6.83-6.818 10.605-15.896 10.632-25.566l.029-17.242c0-9.532-3.87-18.868-10.616-25.613-4.594-4.595-4.594-12.069 0-16.664l26.097-26.103v160.494h-44.146v-31.302zm-33.004 49.306h-16.302l16.302-16.302zm79.282 32.304c0 5.223-4.249 9.471-9.471 9.471h-54.812v-26.775h48.915l12.271 10.302c1.97 1.791 3.097 4.338 3.097 7.002zm6.377-192.695c-6.343 0-11.503-5.155-11.514-11.495v-3.612l6.033-6.035c2.929-2.929 2.929-7.678-.001-10.606-7.914-7.914-7.9-21.113.012-29.022 4.628-4.63 10.937-6.562 16.984-5.834v55.09c0 6.349-5.165 11.514-11.514 11.514zm102.556-158.103c-13.254-5.799-29.142-2.765-39.354 7.445-8.041 8.042-11.416 19.01-10.126 29.505-10.827-1.322-21.797 2.419-29.514 10.135-7.72 7.721-11.465 18.691-10.142 29.523-10.489-1.298-21.461 2.08-29.497 10.116-11.666 11.664-13.745 30.496-4.757 44.377l-42.385 42.395c-10.44 10.442-10.44 27.433.001 37.876 3.954 3.954 6.223 9.424 6.223 15.073l-.029 17.143c-.016 5.668-2.228 10.988-6.234 14.987l-62.26 62.26c-19.714 19.714-52.951 19.685-72.633 0l-66.41-66.409c-19.731-19.733-19.744-52.899 0-72.635l62.25-62.259c5.078-5.078 12.492-6.948 19.472-6.241v33.297c0 14.62 11.895 26.515 26.515 26.515s26.515-11.895 26.515-26.515v-20.007c4.725-1.145 9.062-3.563 12.589-7.089l164.326-164.318 51.982-8.252-18.208 18.596c-2.897 2.96-2.848 7.708.112 10.606 2.896 2.836 7.762 2.791 10.606-.112l17.622-17.997z"
          />
        </g>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root"
      fill="#6D899E"
      focusable="false"
      height="512"
      viewBox="0 0 480 480"
      width="512"
      x="0px"
      xmlns="http://www.w3.org/2000/svg"
      y="0px"
    >
      <g>
        <path
          d="m113.572 338.626c-2.93-2.928-7.678-2.928-10.607 0l-19.88 19.88c-2.929 2.93-2.929 7.678 0 10.607l80.067 80.068c1.407 1.406 3.314 2.196 5.304 2.196s3.896-.79 5.304-2.196l19.881-19.881c1.406-1.407 2.196-3.314 2.196-5.304s-.79-3.896-2.196-5.304zm54.884 94.645-69.461-69.461 9.273-9.273 69.462 69.461z"
        />
        <path
          d="m282.902 243.491-50.435 50.436c-2.929 2.93-2.929 7.678 0 10.607 1.465 1.464 3.385 2.196 5.304 2.196s3.839-.732 5.304-2.196l50.435-50.436c2.929-2.93 2.929-7.678 0-10.607-2.93-2.928-7.678-2.928-10.608 0z"
        />
        <path
          d="m467.067 72.911c.137-4.646-4.131-8.354-8.669-7.629l-76.791 12.19c-1.563.248-3.008.984-4.127 2.104 0 0-167.323 167.181-168.032 167.637v-89.392c0-4.143-3.357-7.5-7.5-7.5h-30.529v-13.805c0-16.834 13.695-30.529 30.529-30.529h22.28l17.389 17.436c10.651 10.679 28.337 10.485 38.747-.442l16.19-16.993h23.689c4.847 0 10.206-.62 10.725-6.729.55-6.493-5.546-8.271-10.725-8.271h-24.109c8.347-8.719 13.492-20.527 13.492-33.522v-29.393c-.001-15.479-12.595-28.073-28.075-28.073h-40.912c-15.48 0-28.074 12.594-28.074 28.073v.024.008 29.36c0 12.727 5.025 24.666 13.467 33.522h-24.084c-25.104 0-45.529 20.425-45.529 45.529v98.506c-10.916-.595-22.223 2.779-30.079 10.635l-62.25 62.258c-25.546 25.538-25.546 68.301-.001 93.849l66.41 66.41c11.509 11.509 27.545 18.605 43.818 19.362.003 13.49 10.979 24.464 24.47 24.464h124.622c21.628 0 33.365-28.555 16.118-42.879l-11.784-9.893v-152.027c17.325 8.387 38.028-4.723 38.028-23.881v-47.445c2.605.859 5.586.259 7.657-1.814 2.866-2.868 2.869-7.734-.005-10.606-7.919-7.911-7.911-21.119.003-29.03 7.907-7.909 21.115-7.909 29.022-.001 2.929 2.93 7.678 2.929 10.606.001 2.867-2.867 2.875-7.733.001-10.606-8.002-8.003-7.997-21.026.01-29.033 7.903-7.905 21.108-7.913 29.011-.003 4.233 4.239 11.78 1.748 12.713-4.125l12.155-76.566c.074-.387.111-.791.123-1.181zm-197.564 39.723c-5.7 4.92-11.455 4.986-17.266.197l-6.825-6.844h30.424zm-19.873-23.652c-13.198-4.796-22.065-17.462-22.065-31.518v-16.01c8.131 4.036 19.806 8.172 33.53 8.172 13.725 0 25.399-4.136 33.53-8.172v16.011c0 22.735-23.692 39.246-44.995 31.517zm-8.991-73.982h40.912c5.978 0 11.021 4.038 12.572 9.526-5.599 3.573-17.897 10.1-33.028 10.1s-27.429-6.527-33.028-10.1c1.551-5.488 6.593-9.526 12.572-9.526zm-46.192 150.321v117.998c0 6.35-5.165 11.515-11.515 11.515s-11.515-5.165-11.515-11.515v-117.998zm-5.069 321.183c12.197-2.242 23.656-7.908 32.812-16.279h31.408v26.775h-54.811c-5.222 0-9.409-5.274-9.409-10.496zm97.224-80.585c6.83-6.818 10.605-15.896 10.632-25.566l.029-17.242c0-9.532-3.87-18.868-10.616-25.613-4.594-4.595-4.594-12.069 0-16.664l26.097-26.103v160.494h-44.146v-31.302zm-33.004 49.306h-16.302l16.302-16.302zm79.282 32.304c0 5.223-4.249 9.471-9.471 9.471h-54.812v-26.775h48.915l12.271 10.302c1.97 1.791 3.097 4.338 3.097 7.002zm6.377-192.695c-6.343 0-11.503-5.155-11.514-11.495v-3.612l6.033-6.035c2.929-2.929 2.929-7.678-.001-10.606-7.914-7.914-7.9-21.113.012-29.022 4.628-4.63 10.937-6.562 16.984-5.834v55.09c0 6.349-5.165 11.514-11.514 11.514zm102.556-158.103c-13.254-5.799-29.142-2.765-39.354 7.445-8.041 8.042-11.416 19.01-10.126 29.505-10.827-1.322-21.797 2.419-29.514 10.135-7.72 7.721-11.465 18.691-10.142 29.523-10.489-1.298-21.461 2.08-29.497 10.116-11.666 11.664-13.745 30.496-4.757 44.377l-42.385 42.395c-10.44 10.442-10.44 27.433.001 37.876 3.954 3.954 6.223 9.424 6.223 15.073l-.029 17.143c-.016 5.668-2.228 10.988-6.234 14.987l-62.26 62.26c-19.714 19.714-52.951 19.685-72.633 0l-66.41-66.409c-19.731-19.733-19.744-52.899 0-72.635l62.25-62.259c5.078-5.078 12.492-6.948 19.472-6.241v33.297c0 14.62 11.895 26.515 26.515 26.515s26.515-11.895 26.515-26.515v-20.007c4.725-1.145 9.062-3.563 12.589-7.089l164.326-164.318 51.982-8.252-18.208 18.596c-2.897 2.96-2.848 7.708.112 10.606 2.896 2.836 7.762 2.791 10.606-.112l17.622-17.997z"
        />
      </g>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
